Maven Build Failure on prospero-standalone-galleon-pack

While running the command mvn clean install -Pdist -DskipTests, the build fails at the prospero-standalone-galleon-pack module with the following error:

[ERROR] Failed to execute goal org.wildfly.galleon-plugins:wildfly-galleon-maven-plugin:7.1.0.Final:build-feature-pack (prospero-galleon-pack-build) on project prospero-standalone-galleon-pack: Failed to resolve org.jboss.logging:logging-parent:pom:1.0.1.Final: The following artifacts could not be resolved: org.jboss.logging:logging-parent:pom:1.0.1.Final (present, but unavailable): Cannot access jboss-public-repository (https://repository.jboss.org/nexus/content/groups/public/) in offline mode and the artifact org.jboss.logging:logging-parent:pom:1.0.1.Final has not been downloaded from it before. -> [Help 1]

--- a/pom.xml
+++ b/pom.xml
@@ -603,7 +603,7 @@
                 <enabled>true</enabled>
                 <updatePolicy>never</updatePolicy>
             </snapshots>
-            <id>jboss-public-repository</id>
+            <id>jboss-public-repository-group</id>
             <name>JBoss Public Repository</name>
             <url>https://repository.jboss.org/nexus/content/groups/public/</url>
             <layout>default</layout>
@@ -619,7 +619,7 @@
                 <enabled>true</enabled>
                 <updatePolicy>never</updatePolicy>
             </snapshots>
-            <id>jboss-public-repository</id>
+            <id>jboss-public-repository-group</id>
             <name>JBoss Public Repository</name>
             <url>https://repository.jboss.org/nexus/content/groups/public/</url>
             <layout>default</layout>

